Automattic/jetpack
# Fix/show payments configuration in small width

Fixes https://github.com/Automattic/wp-calypso/issues/40493

#### Changes proposed in this Pull Request:

When the Payments block is inserted into a containing block that is too narrow (ie, into a column), the `Placeholder` instructions for configuring the block are automatically hidden. This prevents the user from selecting an initial plan and configuring the block correctly, so it is (inexplicably to the user) not rendered on the frontend.

This PR overrides the CSS that hides the Placeholder instructions for the Payments block. This will enable users to actually use the block inside columns/group blocks/etc.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Insert a Columns block into a new post with at least three columns.
* Insert a Payments block into a column.
* Verify that you can see the 'Payments' banner over the block (see screenshots)
* Select a plan (you may need to connect to Stripe first)
* View the post on the frontend and verify that you can see the block.

#### Screenshots
| Before | After |
| --- | --- |
| <img width="661" alt="Screen Shot 2021-06-16 at 4 53 09 PM" src="https://user-images.githubusercontent.com/63313398/122310969-094fc000-cec6-11eb-8928-12bfd919eb41.png"> | <img width="670" alt="Screen Shot 2021-06-16 at 4 52 40 PM" src="https://user-images.githubusercontent.com/63313398/122310977-0ead0a80-cec6-11eb-8b7e-d814aa1d189a.png"> |
